src/qt/scicon.cpp

```diff
@@ -51,6 +56,9 @@ QIcon SingleColorIcon(const QString& filename, const QColor& colorbase)
 
 QColor SingleColor()
 {
+#if defined(WIN32) || defined(MAC_OSX)
+    return QColor(0,0,0);
```

Comment:
I wonder if this will cause code-unreachable warnings, may be better to #else #ifdef the rest of the function, same for `SingleColorIcon`.